roles/sap_ha_install_hana_hsr/tasks/log_mode.yml

```diff
@@ -4,7 +4,10 @@
   become: true
   become_user: "{{ sap_ha_install_hana_hsr_sid | lower }}adm"
   ansible.builtin.shell: |
+    set -e -o pipefail
     grep "log_mode" /usr/sap/{{ sap_ha_install_hana_hsr_sid }}/SYS/global/hdb/custom/config/global.ini | grep normal
+  args:
```

Comment:
Why add args exec declaration when removed elsewhere?

Reply:
Valid question. Researched and removed from all tasks because the commands used should all be POSIX compliant and do not need a bash executable.